python/test/link_TEST.py

```diff
@@ -0,0 +1,103 @@
+            self.post_iterations += 1
+
+        def on_pre_udpate_cb(_info, _ecm):
+            self.pre_iterations += 1
+            world_e = world_entity(_ecm)
+            self.assertNotEqual(K_NULL_ENTITY, world_e)
+            w = World(world_e)
+            m = Model(w.model_by_name(_ecm, 'model_test'))
+            link = Link(m.link_by_name(_ecm, 'link_test'))
+            # Entity Test
+            self.assertNotEqual(K_NULL_ENTITY, link.entity())
+            # Valid Test
+            self.assertTrue(link.valid(_ecm))
+            # Name Test
+            self.assertEqual('link_test', link.name(_ecm))
+            # Parent Model Test
+            self.assertEqual(m.entity(), link.parent_model(_ecm).entity())
+            # Canonical test
+            self.assertTrue(link.is_canonical(_ecm))
+            # Wind Mode test
+            self.assertTrue(link.wind_mode(_ecm))
+            # Collisions Test
+            self.assertNotEqual(K_NULL_ENTITY, link.collision_by_name(_ecm, 'collision_test'))
+            self.assertEqual(1, link.collision_count(_ecm))
+            # Visuals Test
+            self.assertNotEqual(K_NULL_ENTITY, link.visual_by_name(_ecm, 'visual_test'))
+            self.assertEqual(1, link.visual_count(_ecm))
+            # World Pose Test
+            self.assertEqual(None, link.world_pose(_ecm))
+            self.assertEqual(None, link.world_inertial_pose(_ecm))
+            # World Velocity Test
+            self.assertEqual(None, link.world_linear_velocity(_ecm))
+            self.assertEqual(None, link.world_angular_velocity(_ecm))
+            link.enable_velocity_checks(_ecm, True)
+            link.set_linear_velocity(_ecm, Vector3d())
+            link.set_angular_velocity(_ecm, Vector3d())
+            self.assertEqual(Vector3d(), link.world_linear_velocity(_ecm))
+            self.assertEqual(Vector3d(), link.world_angular_velocity(_ecm))
+            # World Acceleration Test
+            self.assertEqual(None, link.world_linear_acceleration(_ecm))
+            self.assertEqual(None, link.world_angular_acceleration(_ecm))
+            link.enable_acceleration_checks(_ecm, True)
+            self.assertEqual(Vector3d(), link.world_linear_acceleration(_ecm))
+            self.assertEqual(Vector3d(), link.world_angular_acceleration(_ecm))
+            # World Inertia Matrix Test
+            self.assertEqual(Matrix3d(1,0,0,0,1,0,0,0,1), link.world_inertia_matrix(_ecm))
+            # World Kinetic Energy Test
+            self.assertEqual(0, link.world_kinetic_energy(_ecm))
+            link.enable_velocity_checks(_ecm, False)
+            link.enable_acceleration_checks(_ecm, False)
+
+
+        def on_udpate_cb(_info, _ecm):
+            self.iterations += 1
+
+        fixture.on_post_update(on_post_udpate_cb)
+        fixture.on_update(on_udpate_cb)
+        fixture.on_pre_update(on_pre_udpate_cb)
+        fixture.finalize()
+
+        server = fixture.server()
+        server.run(True, 1000, False)
```

Comment:
Can we reduce the number of iterations to just 2? Not sure if we need 1000. It 
would reduce the test time significantly.

Reply:
Sure and done.